PROCESS.md

```diff
@@ -0,0 +1,206 @@
+# JSON Schema Specification Development and Publication Process
+
+## Purpose
+
+This document describes the development and publication process for the JSON Schema specifications contained within this repository.
+
+- [JSON Schema Core](./jsonschema-core.md)
+- [JSON Schema Validation](./jsonschema-validation.md)
+
+## Definitions
+
+### Defined Behavior
+
+Some behaviors within JSON Schema may be explicitly or implicitly undefined by the specifications for various reasons.  How to handle these behaviors is generally left to implementations.
+
+A defined behavior is one that is fully and unambiguously defined by the specifications.
+
+An undefined behavior is said to have an "indeterminate" validation result since implementations may resolve the behavior in different ways.
+
+### Stability and Breaking Changes
+
+Stability is defined using the level of compatibility between sequential releases.  If all schemas which are written to one release produce the same defined behavior under the following release, then those releases are compatible, and the specification is said to be stable between them.
+
+If an existing schema under the new release exhibits defined behavior that is contrary to defined behavior under the previous release, the new release is said to contain breaking changes and the specification is unstable between those releases.
+
+If a new release fully defines a previously undefined (or under-defined) behavior, the new release is still considered compatible, even if it contradicts the decision of any particular implementation.
+
+For reference, this table shows the validation results of a hypothetical schema and instance across two consecutive releases to illustrate the compatibility of those releases:
+
+<table>
+  <tr>
+    <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Next➡️<br>⬇️Current</td>
+    <th width="150" align="center">pass</th>
+    <th width="150" align="center">fail</th> 
+    <th width="150" align="center">indeterminate</th> 
+  </tr>
+  <tr>
+    <th align="center">pass</th>
+    <td align="center">✅</td> 
+    <td align="center">❌</td>
+    <td align="center">❌</td>
+  </tr>
+  <tr>
+    <th align="center">fail</th>
+    <td align="center">❌</td> 
+    <td align="center">✅</td>
+    <td align="center">❌</td>
+  </tr>
+  <tr>
+    <th align="center">indeterminate</th>
+    <td align="center">✅</td> 
+    <td align="center">✅</td> 
+    <td align="center">✅</td>
+  </tr>
+</table>
```

Comment:
This works in GH and our spec markdown setup. The only thing it can't do is fixed column width, but I don't think that's a big deal.

```suggestion
| _Next_ ➡️<br>⬇️ _Current_ | pass | fail | indeterminate |
|:---:|:---:|:---:|:---:|
| **pass** | ✅ | ❌ | ❌ |
| **fail** | ❌ | ✅ | ❌ |
| **indeterminate** | ✅ | ✅ | ✅ |
```

Reply:
<table>
  <tr>
    <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Next➡️<br>⬇️Current</td>
    <th width="150" align="center">pass</th>
    <th width="150" align="center">fail</th> 
    <th width="150" align="center">indeterminate</th> 
  </tr>
  <tr>
    <th align="center">pass</th>
    <td align="center">✅</td> 
    <td align="center">❌</td>
    <td align="center">❌</td>
  </tr>
  <tr>
    <th align="center">fail</th>
    <td align="center">❌</td> 
    <td align="center">✅</td>
    <td align="center">❌</td>
  </tr>
  <tr>
    <th align="center">indeterminate</th>
    <td align="center">✅</td> 
    <td align="center">✅</td> 
    <td align="center">✅</td>
  </tr>
</table>

---

| _Next_ ➡️<br>⬇️ _Current_ | pass | fail | indeterminate |
|:---:|:---:|:---:|:---:|
| **pass** | ✅ | ❌ | ❌ |
| **fail** | ❌ | ✅ | ❌ |
| **indeterminate** | ✅ | ✅ | ✅ |

---

I kinda like it better with the column widths, but I do like italicizing the corner text.

Reply:
I think it's not a good idea to use HTML in markdown unless there's a really good reason, and I don't think it's necessary in this case. For example, here's how you're comment rendered for me using a smaller screen. I'm sure the HTML could be tweaked to fix this problem, but we're creating more maintenance work for ourselves.

![image](https://github.com/json-schema-org/json-schema-spec/assets/716571/9772932e-109a-4cec-bcd9-b169f291a571)